.github/workflows/testing.yml

```diff
@@ -50,8 +50,11 @@ jobs:
           ./testing/scripts/yarn-install.sh
           ./testing/scripts/yarn-lockfile-check.sh
 
-      - name: Lint checking
-        run: ./testing/scripts/lint-check.sh
```

Comment:
This was passing even with a non-zero exit currently https://github.com/mdn/yari/runs/1560304722?check_suite_focus=true#step:10:17
I didn't delete the bash file though

Reply:
Oh I see. The `link-check.sh` didn't have the `set -e` or something?
EIther way, I much prefer your change. Can you please also `git rm testing/scripts/lint-check.sh` please?